nin/frontend/app/scripts/controllers/main.js

```diff
@@ -78,7 +78,8 @@ class MainCtrl {
             try {
               let node = demo.nm.createNode(nodeInfo);
               demo.nm.insertOrReplaceNode(node);
-            } catch (e) {
+            } catch(e) {
+              console.error(e);
```

Comment:
Will this flood the logs with the same error 5 times a second? Should we differentiate between "node hasn't been loaded yet" vs "node crashed when we tried to init it"?